docs/governance/templates/release_issue.md

```diff
@@ -26,43 +26,55 @@ There are 3 types of releases:
 1. Close the release issue.
 
 !-->
+
+Create a Branch
+---------------
+**These instructions are for the first release candidate of a minor (X.Y) release.**
+- [ ] Create a branch in the upstream repository. It should be named release-X.Y. Example: release-0.5. At this point there's effectively a code freeze for this version and all work on master will be included in a future version.
+
+Create New Version
+------------------
+_Find the release branch relevant for your release. It will have the name release-X.Y. While doing a release in the branch no changes should be submitted outside of the process._
+
+- [ ] Announce a PR freeze on release-X.Y branch on [open-match-discuss@](mailing-list-post).
+- [ ] Create a PR to bump the version.
+  - [ ] Open the [`Makefile`](makefile-version) and change BASE_VERSION value. Release candidates use the -rc.# suffix.
+  - [ ] Open the [`install/helm/open-match/Chart.yaml`](om-chart-yaml-version) and [`install/helm/open-match-example/Chart.yaml`](om-example-chart-yaml-version) and change the `appVersion` and `version` entries.
+  - [ ] Open the [`install/helm/open-match/values.yaml`](om-values-yaml-version) and [`install/helm/open-match-example/values.yaml`](om-example-values-yaml-version) and change the `tag` entries.
+  - [ ] Open the [`site/config.toml`]  and change the `release_branch` and `release_version` entries.
+  - [ ] Open the [`README.md`](readme-deploy) update the version references.
+  - [ ] Run `make clean release`
+  - [ ] There might be additional references to the old version but be careful not to change it for places that have it for historical purposes.
+  - [ ] Submit the pull request.
+
 Complete Milestone
 ------------------
 - [ ] Create the next version milestone, use [semantic versioning](https://semver.org/) when naming it to be consistent with the [Go community](https://blog.golang.org/versioning-proposal).
 - [ ] Visit the [milestone](https://github.com/GoogleCloudPlatform/open-match/milestone).
-  - [ ] Open a document for a draft [release notes](release.md).
-  - [ ] Add the milestone tag to all PRs and issues that were merged since the last milestone. Look at the [releases page](https://github.com/GoogleCloudPlatform/open-match/releases) and look for the "X commits to master since this release" for the diff. The link resolves to, https://github.com/GoogleCloudPlatform/open-match/compare/v{version}...master.
+  - [ ] Create a *draft* release with the [release template][release-template].
+    - [ ] The value of `tag` should be the release version with format: v{version}. Example: v0.5.0. Append -rc.# for release candidates. Example: v0.5.0-rc.1.
```

Comment:
Maybe call out specifically here that this is the PR submitted to your release branch?